test/e2e/common/util.go

```diff
@@ -19,13 +19,24 @@ import (
 	. "github.com/onsi/gomega"
 )
 
+const NSE2EPrefix = "e2e-maya"
+const NSDefault = "default"
 const NSMayastor = "mayastor"
 const CSIProvisioner = "io.openebs.csi-mayastor"
+const DefaultVolumeSize = 64
```

Comment:
Nit: Might be nice to include the units in the name i.e. DefaultVolumeSizeMb

Reply:
Will do, I spotted another hard coded setting in RunFio, will fix both.

Reply:
Fixed now